projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -87,9 +87,20 @@ <h5 class="d-lg-none align-self-end">
             </div>
             <div class="col d-none d-lg-block">
               <ng-container *ngIf="policy.paymentMethod; else NA">
-                {{ policy.paymentMethod.cardType?.code | uppercase }}
-                {{ policy.paymentMethod?.cardNumber }}
+                <ng-container
+                  *ngIf="
+                    policy.paymentMethod?.cardType?.code;
+                    else invoicePaymentMethod
+                  "
+                >
+                  {{ policy.paymentMethod.cardType?.code | uppercase }}
+                  {{ policy.paymentMethod?.cardNumber }}
+                </ng-container>
```

Comment:
Maybe this whole section of code can be inside one ng-template? This code is again repeating.